source/js/components/search-form.js

```diff
@@ -1,98 +1,106 @@
+                    ${item.specialties.map(item => `<li class="specialties-list__item">${item}</li>`).join('')}
                 </ul>`
     }
 
-    function renderTotalNumberOfResults(data) {
-        const totalNumbers = data.length;
-        const totalResults =  `<h2 class="doctors-total-result">Total Results: <span>${totalNumbers}</span></h2>`
-        layoutEl.insertAdjacentHTML('afterbegin', totalResults);
-    }
 
-    function renderDoctorsList(data) {
-        const renderDoctors = data.map(item => {
-            return `<li class="doctor-list__item"> 
-                        <div class="doctor-info">
-                            <div class="doctor-info__general">
-                                <div class="doctor-image">
-                                    <img src="${item.image}" alt="">
-                                </div>
-                                <div class="doctor-general">
-                                    <h3 class="doctor-name"><a href="${item.url}" target="_blank">${item.fullName}</a></h3>
-                                    ${renderSpecialties(item)}
-                                </div>
-                            </div>
-                            <div class="doctor-info__location">
-                                ${renderLocations(item)}
-                            </div>
-                        </div>
-                    </li>`;
-        }).join('');
-
-        layoutEl.insertAdjacentHTML('beforeend', `<ul class="doctor-list">${renderDoctors}</ul>`);
+
+    function renderDoctorsSearchResult(data) {
+        const totalNumberOfDoctors = data.length;
         
+        return `<h2 class="doctors-total-result">Total Results: <span>${totalNumberOfDoctors}</span></h2>
+                <ul class="doctor-list">
+                    ${data.map(item => 
+                        `<li class="doctor-list__item"> 
+                                <div class="doctor-info">
+                                    <div class="doctor-info__general">
+                                        <div class="doctor-image">
+                                            <img src="${item.image}" alt="">
+                                        </div>
+                                        <div class="doctor-general">
+                                            <h3 class="doctor-name"><a href="${item.url}" target="_blank">${item.fullName}</a></h3>
+                                            ${renderSpecialties(item)}
+                                        </div>
+                                    </div>
+                                    <div class="doctor-info__location">
+                                        ${renderLocations(item)}
+                                    </div>
+                                </div>
+                            </li>`
+                    ).join('')}
+                </ul>`;        
     }
 
     function renderResults(data) {
         layoutEl.innerHTML = '';
```

Comment:
[fix] With how you're doing the next line now, this line is unnecessary.

Reply:
yep Fixed.